### ISSUE TITLE
Question regarding training specifications

### ISSUE THREAD

**Issue by watermeleon** [carolius/Influence-benchmark#179]
Hi there,

As mentioned in [my response to issue #178](https://github.com/marcus-jw/Targeted-Manipulation-and-Deception-in-LLMs/issues/178#issuecomment-3606838200), I understand most aspects of the repository but have some questions about the RL training process.

Specifically, I'd like to clarify how much data is used to train the models (for example, the fully vulnerable/gameable therapy-talk model). The paper mentions 76.8k initial states, and based on the paper and repo, here's what I understand happens:

Per iteration:
1. 160 random initial states are selected and a single response/trajectory is generated for each
2. The top 1/16th (10 samples) receive "thumbs-up" and the bottom 1/16th (10 samples) receive "thumbs-down"
3. One round of KTO optimization is performed on these 20 samples

From the hyperparameters (e.g., in [`experiment_configs/therapy-talk/_base_therapy.yaml`](https://github.com/marcus-jw/Targeted-Manipulation-and-Deception-in-LLMs/blob/627e1f0b08650b0e58481d9289b60c263b48df24/targeted_llm_manipulation/config/experiment_configs/therapy-talk/_base_therapy.yaml#L24)), it looks like you run about 20 iterations. 

**Q1**. Does this mean the total training consists of **20 samples/iteration × 20 iterations = 400 total samples** used for optimization/update steps? Just want to make sure I'm interpreting the training setup correctly.

**Q2**. Also am I getting right what you mean by "iteration"? I was a bit confused, as other sources (such as TRL), often speak of "steps" or "epochs". 

**Q3**. How does your learning-rate over training look like? I see that you don't use a standard scheduler (since `lr_scheduler_type: "constant"`), but you do set ``across_iter_lr_mult_factor: 0.9`, from which I gather that after each iteration, the learning rate gets reduced by 10%.

**Q4**. If you have any Weights & Biases (wandb) logs you could share, that would be fantastic as well!

Thanks for your time and for the excellent paper. It would be great if you could answer some of the questions, as I think additional clarity on the training setup would be helpful for others in the community too.

Kind regards,  
Leon
